src/Orleans/Telemetry/Consumers/TraceTelemetryConsumer.cs

```diff
@@ -37,12 +36,21 @@ public void TrackTrace(string message, Severity severity)
                 case Severity.Off:
                     return;
             }
-            Trace.Flush();
```

Comment:
I don't know if you missed the other files but, both flush and close are there and are being called according with new implementation...